_docs/guides/forseti-security-best-practices.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Forseti Security Best Practices 
+order: 004
+---
+# {{ page.title }}
+
+## Service Accounts
+When you use multiple service accounts with your Forseti Security deployments,
```

Comment:
I'm not so sure that it's only the multiple service accounts that implements least privilege -- we also must observe the principle of assigning the lowest role necessary for the task. Pinging @ahoying for guidance on how to word this.